data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -357,6 +357,7 @@ tourism-viewpoint                                   # icon z14- (also has captio
 
 amenity-place_of_worship-buddhist                   # icon z14- (also has caption(optional) z14-, area z16-)
 amenity-place_of_worship-christian                  # icon z14- (also has caption(optional) z14-, area z16-)
+amenity-place_of_worship-christian-mormon           # icon z14- (also has caption(optional) z14-, area z16-)
```

Comment:
`[amenity=place_of_worship][religion=christian]` icon is declared z14- hence with the current changes there will be a cross icon at z14 and then it'll change to a new icon starting at z15-.

So either need to add a small icon for z14, or maybe demote all places of worship to z15-?

I think they were put to z14- before because many of temples are touristic attractions, but recently we decided to rely on them being tagged as a tourism=attraction (#5662 ), so "everyday" non-touristic temples could be made less visible.
E.g. many touristic attractions like tourism-artwork, historic-memorial have a lower z15- visibility now.
But it needs some discussion before.
CC @biodranik

Reply:
Added small icon